discovery-provider/plugins/pedalboard/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
probably could be shared with some others we have

https://github.com/AudiusProject/audius-protocol/blob/main/identity-service/tsconfig.json